config.default.yaml

```
@@ -94,6 +94,8 @@ build_shape_options:
   worldpop_method: "standard"  # "standard" pulls from web 1kmx1km raster, "api" pulls from API 100mx100m raster, false (not "false") no pop addition to shape which is useful when generating only cutout
   gdp_method: "standard"  # "standard" pulls from web 1x1km raster, false (not "false") no gdp addition to shape which useful when generating only cutout
   contended_flag: "set_by_country" # "set_by_country" assigns the contended areas to the countries according to the GADM database, "drop" drops these contended areas from the model
+  gadm_file_prefix_v41: "gadm41_"
+  gadm_url_prefix_v41: "https://geodata.ucdavis.edu/gadm/gadm4.1/gpkg/"
```

perhaps we can keep that generic, for easy future updates (also meeting the argument names of the gdam functions):
```suggestion
  gadm_file_prefix: "gadm41_"
  gadm_url_prefix: "https://geodata.ucdavis.edu/gadm/gadm4.1/gpkg/"
```